selfdrive/car/toyota/interface.py

```diff
@@ -29,7 +30,7 @@ def _get_params(ret, candidate, fingerprint, car_fw, disable_openpilot_long, exp
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_TOYOTA_ALT_BRAKE
 
     if ret.flags & ToyotaFlags.SECOC.value:
-      ret.secOcRequired = True
+      # ret.secOcRequired = True
```

Comment:
Not sure what this was for. Doesn't look like any code used it when I did a find-in-all-files search for `secOcRequired` so I've just commented it out to get FrogPilot running.